app.js

```diff
@@ -1,83 +1,77 @@
+/*eslint no-undef: "error"*/
+/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
+/*eslint-env node*/
+
 const express = require('express');
 const app = express();
-const morgan = require('morgan');
 const bodyParser = require('body-parser');
+const alasql = require('alasql');
+
 
 app.use(express.static('public'));
 
-const PORT = process.env.PORT || 4001;
-
-const items = [
-  {
-    id: 1,
-    name: 'Bread',
-    quantity: '1',
-    unit: 'Kg'
-  },
-  {
-    id: 2,
-    name: 'Milk',
-    quantity: '1',
-    unit: 'L'
-  },
-  {
-    id: 3,
-    name: 'Pasta',
-    quantity: '2',
-    unit: 'Kg'
-  }
-];
-let nextId = 4;
+const PORT = process.env.PORT || 4002;
```

Comment:
consider moving configuration into JSON files, this module is standard https://www.npmjs.com/package/config

Reply:
Done as a port of new changeset.